editor/tests/org.fusesource.ide.camel.editor.tests.integration/src/main/java/org/fusesource/ide/camel/editor/integration/globalconfiguration/dataformat/wizards/NewDataFormatWizardIT.java

```diff
@@ -135,4 +212,80 @@ private void checkSpecialParameterLoaded(final String id, AbstractCamelModelElem
 			Assertions.assertThat(libraryParameter).isEqualTo("Gson");
 		}
 	}
+
+	@Before
+	public void setup() throws Exception {
+		CamelEditorUIActivator.pluginLog().logInfo("Starting setup for "+ NewDataFormatWizardIT.class.getSimpleName());
```

Comment:
a PR should take care of no adding duplicated code, so this should be considered for this PR. You can decide to take the risk to maintain 2 parts of duplicated code but it needs to be considered in this PR